### ISSUE TITLE
sub_mean normalization

### ISSUE THREAD

**Issue by Podidiving** [divamgupta/image-segmentation-keras#91]
I'm currently reading the code and cannot understand, why we should reverse channels in the image, when using sub_mean normalization. It is in `data_loader.py`

**Comment by divamgupta** [comment]
OpenCV loads the image in BGR rather than RGB.

**Comment by rola93** [comment]
Hello! It's still strange for me... it looks like images in `get_image_array` are returned in RGB or BGR depending on `imgNorm` variable, rigth? (assuming `image_input` is the path to an image file).

As I understand, images are read as BGR with OpenCV by default, then it is never converted to RGB except for the case `imgNorm == "sub_mean"`, is this correct?

Can you explain the logic behind this? I'm trying to understand how should I provide images to the model straigthforward (i.e. without the logic provided by the library), so, I need to understand if I need RGB or BGR and how to pre-process it properly, this second point is specially important for the pretrained models given.

In particular, I'm interested in `keras_segmentation.models.pspnet.pspnet_50` model.

Thanks in advance
